packages/main-api/src/controllers/v1/supports/mod.rs

```diff
@@ -0,0 +1,53 @@
+use bdk::prelude::*;
+use by_axum::{
+    auth::Authorization,
+    axum::{Extension, Json, extract::State, routing::post},
+};
+use dto::*;
+
+#[derive(Clone, Debug)]
+pub struct SupportControllerV1 {
```

Comment:
```suggestion
pub struct SupportController {
```

remove version in controller name because module path already contains version.